Project.toml

```
@@ -1,7 +1,7 @@
 name = "PDMatsExtras"
 uuid = "2c7acb1b-7338-470f-b38f-951d2bcb9193"
 authors = ["Invenia Technical Computing"]
-version = "2.2.0"
+version = "2.2.1"
```

```suggestion
version = "2.3.0"
```
Adding new functionality bumps a minor version